LogFile.cs

```diff
@@ -65,7 +71,7 @@ public override void WriteDebug(string message)
                 CheckDirectory();
                 if (message != String.Empty)
                 {
-                    message = String.Format("{0,-29} - [D] {1}", DateTime.UtcNow.ToString(DateTimeFormat), message);
+                    message = String.Format("{0,-18} - [D] {1}", DateTime.UtcNow.ToString(DateTimeFormat), message);
```

Comment:
Hi Dave! I changed it back to its original value. I was just doing some testing back then. Thanks!